installation/sql/mysql/localise.sql

```diff
@@ -67,3 +65,22 @@ UPDATE IGNORE `#__modules` SET `title` = 'Joomla Version' WHERE `id` = 86;
 UPDATE IGNORE `#__modules` SET `title` = 'Beispieldaten' WHERE `id` = 87;
 UPDATE IGNORE `#__modules` SET `title` = 'Letzte Aktionen' WHERE `id` = 88;
 UPDATE IGNORE `#__modules` SET `title` = 'Datenschutz Dashboard' WHERE `id` = 89;
+UPDATE IGNORE `#__modules` SET `title` = 'Anmeldung Support' WHERE `id` = 90;
+UPDATE IGNORE `#__modules` SET `title` = 'System Dashboard' WHERE `id` = 91;
+UPDATE IGNORE `#__modules` SET `title` = 'Inhalt Dashboard' WHERE `id` = 92;
+UPDATE IGNORE `#__modules` SET `title` = 'Menü Dashboard' WHERE `id` = 93;
+UPDATE IGNORE `#__modules` SET `title` = 'Erweiterungen Dashboard' WHERE `id` = 94;
+UPDATE IGNORE `#__modules` SET `title` = 'Benutzer Dashboard' WHERE `id` = 95;
+UPDATE IGNORE `#__modules` SET `title` = 'Beliebte Beiträge' WHERE `id` = 96;
+UPDATE IGNORE `#__modules` SET `title` = 'Kürzlich hinzugefügte Beiträge' WHERE `id` = 97;
+UPDATE IGNORE `#__modules` SET `title` = 'Angemeldete Benutzer' WHERE `id` = 98;
+UPDATE IGNORE `#__modules` SET `title` = 'Frontend Link' WHERE `id` = 99;
```

Comment:
```suggestion
UPDATE IGNORE `#__modules` SET `title` = 'Website Link' WHERE `id` = 99;
```
Wir nutzen leider ja noch nicht "Frontend".